### PR TITLE
Fix for issue #369

### PR DESCRIPTION
**Pull request by HgAlexx**

### CODE REVIEW

File: src/react-sortable-tree.js [hunk 1]
```diff
@@ -155,10 +155,12 @@ class ReactSortableTree extends Component {
     const { instanceProps } = prevState;
     const newState = {};
 
+    let isEqualResult = isEqual(instanceProps.treeData, nextProps.treeData);
```
Comment (wuweiweiwu) [inline src/react-sortable-tree.js]: nit: `const`

Comment (wuweiweiwu) [reply src/react-sortable-tree.js]: Can we also name it something like `isTreeDataEqual`?

Comment (wuweiweiwu) [inline src/react-sortable-tree.js]:
```suggestion
    const isTreeDataEqual = isEqual(instanceProps.treeData, nextProps.treeData);
```

File: src/react-sortable-tree.js [hunk 2]
```diff
@@ -155,10 +155,12 @@ class ReactSortableTree extends Component {
...
+
     // make sure we have the most recent version of treeData
     instanceProps.treeData = nextProps.treeData;
 
-    if (!isEqual(instanceProps.treeData, nextProps.treeData)) {
+    if (!isEqualResult) {
```
Comment (wuweiweiwu) [inline src/react-sortable-tree.js]:
```suggestion
    if (!isTreeDataEqual) {
```

File: src/react-sortable-tree.js (hunk 1 above)
Comment (HgAlexx) [reply src/react-sortable-tree.js]: Of course!

**Pull request merged**